### PR TITLE
cura: update to 3.6.0

### PR DESCRIPTION
**Pull request by Gigahawk**

### CODE REVIEW

**Comment by Gigahawk** [comment]
I've had to add `python3-sip-devel` to `makedepends` on libArcus to get it to cross compile.
FindSIP.cmake has gone through significant changes so the previous patch won't work anymore. 

I'm pretty sure the better solution would be to add `/usr/lib/python3.6m` to `PYTHON_INCLUDE_DIRS` but I'm not too familiar with using cmake. Adding `-DPYTHON_INCLUDE_DIRS=/usr/lib/python3.6m` to `configure_args` didn't seem to actually set it.

**Comment by Gigahawk** [comment]
Looks like it crashes when starting, seems like it has something to do with PyQt5

```
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/UM/Qt/Bindings/MainWindow.py", line 76, in __init__
    self.closing.connect(self.preClosing)
TypeError: connect() failed between closing(QQuickCloseEvent*) and preClosing()
```

**Comment by Gigahawk** [comment]
Looks like it crashes when starting, seems like it has something to do with PyQt5

```
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/UM/Qt/Bindings/MainWindow.py", line 76, in __init__
    self.closing.connect(self.preClosing)
TypeError: connect() failed between closing(QQuickCloseEvent*) and preClosing()
```

**Comment by Gigahawk** [comment]
Added a patch to remove the line its crashing on, seems to work fine now.

**Comment by karl-nilsson** [comment]
@Gigahawk v3.6.0 was just released. Let me know if you need any help.

**Comment by Gigahawk** [comment]
Will try to get it up in a few hours, should I get rid of the commits for 3.5.1 or just commit on top?

**Comment by karl-nilsson** [comment]
Edit your commits with an interactive git rebase (git rebase -i). Then, once you're done, force push
Let me know if you have any questions.

**Comment by Gigahawk** [comment]
Seems to work fine, haven't tested any new 3.6 features but slicing works and the bug with large stl files not being read properly is gone.

File: srcpkgs/cura-fdm-materials/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Template file for 'cura-fdm-materials'
 pkgname=cura-fdm-materials
-version=3.4.1
+version=3.6.0
 revision=1
 noarch=yes
-build_style=cmake
 wrksrc="fdm_materials-${version}"
+build_style=cmake
```
Comment (abenson) [inline srcpkgs/cura-fdm-materials/template]: why did this line get moved down

Comment (Gigahawk) [reply srcpkgs/cura-fdm-materials/template]: xlint complained about it

Comment (Gigahawk) [reply srcpkgs/cura-fdm-materials/template]: https://travis-ci.org/void-linux/void-packages/jobs/456307671

**Pull request merged**